lib/streamlit/type_util.py

```diff
@@ -512,6 +520,8 @@ def ensure_iterable(obj: Union[DataFrame, Iterable[V_co]]) -> Iterable[Any]:
     """
     if is_snowpark_data_object(obj):
         obj = convert_anything_to_df(obj)
+    elif is_type(obj, _PYSPARK_DF_TYPE_STR) and callable(getattr(obj, "toPandas")):
```

Comment:
nit: see comment above, this could be both merged into a utils method.

Reply:
It's updated, thank you 👍